invenio_app_ils/ui/src/invenio_app_ils/common/components/ESSelector/ESSelectorLoanRequest.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Form, Header, Modal, Segment } from 'semantic-ui-react';
+import { ESSelector } from './';
+import './ESSelector.scss';
+import { DateTime } from 'luxon';
+import { invenioConfig } from '../../config';
+import { toShortDate, toUTCShortDate } from '../../api/date';
+import { DateRangePicker } from '../DateRangePicker';
+import { isEmpty } from 'lodash';
+
+export const PatronSearchInputContext = React.createContext('false');
```

Comment:
Interesting, but I am not sure I understand when this should be `true` or `false`